community/sustainability_effort.md

```diff
@@ -0,0 +1,58 @@
+# Argo Workflows Sustainability Effort
+
+Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. 
+
+We are trying an Incentive System in which contributors pledge a certain number of hours per week (average) to the project and in return have their PRs prioritized for review.
+
+## Where is help needed?
+
+Help is needed for:
+* [reviewing PRs](#reviewing-PRs)
+* [triaging](../docs/CONTRIBUTING.md#triaging-bugs) new incoming bugs by prioritizing them with `P0`, `P1`, `P2`, and `P3` labels
+* responding to questions in [Github Discussions](https://github.com/argoproj/argo-workflows/discussions)
+* responding to questions in [CNCF Slack](https://argoproj.github.io/community/join-slack) in the `#argo-workflows` and `#argo-wf-contributors` channels 
+
+## Commitment
+
+Argo Workflows has roles as described in the [Contributing](../docs/CONTRIBUTING.md#roles) document.
+Any code contributor in a designated role or with an open Membership request can sign up to participate in this Incentive System.
+
+Participants are expected to try to advance in roles.
+There are different expectations depending on the role:
+- Member and below: should average a minimum of 8 hours per week of either [authoring PRs](../docs/CONTRIBUTING.md#authoring-prs) or performing any activities listed above in the ["Where is help needed?"](#where-is-help-needed) section
```

Comment:
"Member and non-members" or "maintainers"

Reply:
I am going to go with the former since "maintainers" is not a clear concept in the [membership guide](https://github.com/argoproj/argoproj/blob/main/community/membership.md)

Reply:
https://github.com/argoproj/argoproj/blob/main/community/GOVERNANCE.md#roles-and-membership

Reply:
>For the purpose of governance, reviewers, approvers and leads are considered maintainers. The list of current maintainers, their project and organizational affiliation(s)is maintained in the [MAINTAINERS](https://github.com/argoproj/argoproj/blob/master/MAINTAINERS.md) file.

Reply:
good to know, but I think this is more clear, so if you're okay with it, I'll just keep it as is

Reply:
Sure

Reply:
I did small modification to make this clearer. https://github.com/argoproj/argoproj/pull/268